### PR TITLE
Cs fixer

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: template/hooks/php/cs-fixer/fix-cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -u
+. "$GIT_DIR/hooks/git_config_wrapper.sh"
+
+fix_cs()
+{
+	get_hook_config php-cs-fixer executable executable optional php-cs-fixer
+	$executable fix --verbose
```
Comment (Soullivaneuh) [inline template/hooks/php/cs-fixer/fix-cs]: With PHP-CS-Fixer v2, you can specify file to fix to make it quicker: https://github.com/FriendsOfPHP/PHP-CS-Fixer/tree/v2.0.0-alpha#using-php-cs-fixer-on-ci

Comment (greg0ire) [inline template/hooks/php/cs-fixer/fix-cs]: Thanks for the tip @Soullivaneuh , I'll try that!

Comment (greg0ire) [inline template/hooks/php/cs-fixer/fix-cs]: … oh it's alpha. I'll try that after then!

Comment (Soullivaneuh) [inline template/hooks/php/cs-fixer/fix-cs]: Yeah sadly. :-/

File: template/hooks/php/cs-fixer/pre-commit [hunk 2]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+. "$GIT_DIR/hooks/change_detector.sh"
+
+if has_changed pre-commit ./**/*.php ./*.php
```
Comment (greg0ire) [inline template/hooks/php/cs-fixer/pre-commit]: I was confused when I saw this work in unit tests, but it turns out it does not in some cases. I have to rewrite this.

**Pull request merged**